mscore/editpitch.cpp

```diff
@@ -68,13 +68,15 @@ void EditPitch::accept()
       int   col = tableWidget->currentColumn();
       int   row = tableWidget->currentRow();
 
-      // topmost row contains notes for 9-th MIDI octave (numbered as '8')
-      done( (9 - row)*12 + col);
+      // topmost row contains notes for 10-th MIDI octave (numbered as '9')
+      int pitch = (10 - row) * 12 + col;
+      done((pitch > 127)? 127: pitch);
       }
 
 void EditPitch::on_tableWidget_cellDoubleClicked(int row, int col)
       {
-      done( (9 - row)*12 + col);
+      int pitch = (10 - row) * 12 + col;
+      done((pitch > 127)? 127: pitch);
```

Comment:
shouldn't happen, with those fields above G9/127 being disabled and unselectable, but better safe than sorry...